lib/responder.ts

```diff
@@ -52,14 +54,14 @@ export class Responder {
   //todo: overload this function
   sendAck(timeoutMs: any, responsesRemaining?: any, cb?: any): void {
     if (!cb) {
-      cb = _.last(arguments);
-      if (!_.isFunction(cb)) cb = null;
+      cb = last(arguments);
+      if (!isFunction(cb)) cb = null;
```

Comment:
It is, but they are checking if the second argument is too. So parameters number and types are not concrete.